### PR TITLE
Niall dev

### PR DESCRIPTION
**Pull request by NiallEgan**
Added:
- Basic profile page
- The ability to edit observations

### CODE REVIEW

File: app/Http/Controllers/ObservationController.php [hunk 1]
```diff
@@ -119,7 +118,19 @@ public function show(Observation $observation)
      */
     public function edit(Observation $observation)
     {
-        //
+
+      $user = Auth::user();
+      // TODO: How easy is it to spoof user ids?
```
Comment (tjiagoM) [inline app/Http/Controllers/ObservationController.php]: Well, the fact that we are using Auth::user() and related Laravel controls to check the user who is with login, means that probably that should not be a big problem anyway :)

File: resources/views/observation/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+
+                        @if ($errors->has('sector'))
+                            <span class="help-block">
+                                <strong>{{ $errors->first('sector') }}</strong>
+                            </span>
+                        @endif
+                    </div>
+                </div>
+
+                <div>{{{ $errors }}}</div>
```
Comment (tjiagoM) [inline resources/views/observation/edit.blade.php]: I think this was my mistake, but we should remove this line :p

**Comment by tjiagoM** [comment]
I don't think having a specific view for editing the observation is good... It's clearly code duplication so everytime we need to change something it's more difficult to maintain as we have to modify on both sides :/ How easy would it be to put things on the main observation creation view?

File: app/Http/Controllers/ObservationController.php (hunk 1 above)
Comment (NiallEgan) [reply app/Http/Controllers/ObservationController.php]: Okay sounds good, wasn't entirely sure that they were always authenticated by Laravel but apparently they are (https://laravel.com/docs/5.6/requests#cookies)

**Comment by NiallEgan** [comment]
Okay will merge the two together!

**Comment by NiallEgan** [comment]
Looks good. FYI the approach that I was going for with the create_edit form is here: https://stackoverflow.com/questions/22844022/laravel-use-same-form-for-create-and-edit
I think that there will probably be a problem with flashing in the old data on return from errors etc. until we change all of the form elements to use the form library and also include the correct opening method, but I can come back to that later :)

**Comment by tjiagoM** [comment]
Cool, thanks!

Yes, I agree we should change that in the future. Laravel will probably still give us flexibility to control where to show the errors, and also for the specific "tricks" we are using to correctly show the sectors plugin. 

I hope :p

**Pull request merged**